content/en/docs/reference/scheduling/config.md

```diff
@@ -112,73 +112,29 @@ desired.
-  volume limits can be satisfied for the node.
-  Extension points: `filter`.
+|Plugin|Description|Extension Points|
+|---|---|---|
+|`ImageLocality`|Favors nodes that already have the container images that the Pod runs.|`score`|
+|`TaintToleration`|Implements [taints and tolerations](/docs/concepts/scheduling-eviction/taint-and-toleration/).|`filter`, `preScore`, `score`|
+|`NodeName`|Checks if a Pod spec node name matches the current node.|`filter`|
+|`NodePorts`|Checks if a node has free ports for the requested Pod ports.|`preFilter`, `filter`|
+|`NodeAffinity`|Implements [node selectors](/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) and [node affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).|`filter`, `score`|
+|`PodTopologySpread`|Implements [Pod topology spread](/docs/concepts/scheduling-eviction/topology-spread-constraints/).|`preFilter`, `filter`, `preScore`, `score`|
+|`NodeUnschedulable`|Filters out nodes that have `.spec.unschedulable` set to true. |`filter`|
+|`NodeResourcesFit`|Checks if the node has all the resources that the Pod is requesting. The score can use one of three strategies: `LeastAllocated` (default), `MostAllocated` and `RequestedToCapacityRatio`.|`preFilter`, `filter`, `score`.|
+|`NodeResourcesBalancedAllocation`|Favors nodes that would obtain a more balanced resource usage if the Pod is scheduled there.|`score`|
+|`VolumeBinding`|Checks if the node has or if it can bind the requested {{< glossary_tooltip text="volumes" term_id="volume" >}}. The `score` extension point is enabled when `VolumeCapacityPriority` feature is enabled. It prioritizes the smallest PVs that can fit the requested volume size.|`preFilter`, `filter`, `reserve`, `preBind`, `score`|
+|`VolumeRestrictions`|Checks that volumes mounted in the node satisfy restrictions that are specific to the volume provider.|`filter`|
+|`VolumeZone`|Checks that volumes requested satisfy any zone requirements they might have.|`filter`|
+|`NodeVolumeLimits`| Checks that CSI volume limits can be satisfied for the node.|`filter`|
+|`EBSLimits`| Checks that AWS EBS volume limits can be satisfied for the node.|`filter`|
+|`GCEPDLimits`| Checks that GCP-PD volume limits can be satisfied for the node.|`filter`|
+|`AzureDiskLimits`|Checks that Azure disk volume limits can be satisfied for the node.|`filter`|
+|`InterPodAffinity`| Implements [inter-Pod affinity and anti-affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity).|`preFilter`, `filter`, `preScore`, `score`|
+|`PrioritySort`|Provides the default priority based sorting.|`queueSort`|
+|`DefaultBinder`|Provides the default binding mechanism.|`bind`|
+|`DefaultPreemption`| Provides the default preemption mechanism.|`postFilter`|
+|`CinderLimits`|Checks that [OpenStack Cinder](https://docs.openstack.org/cinder/) volume limits can be satisfied for the node.|`filter`|
```

Comment:
If using a table, I recommend using HTML and `rowspan`, with one table cell for each extension point (rather than the comma-separated list that this PR is proposing).

How does that sound?

---

I would also sort the list alphabetically and add a column to show whether the plugin is enabled by default.

Reply:
It sounds good to me however let me test out first.  Thanks